src/lib/import_export/csv_parser.cpp

```diff
@@ -40,7 +40,7 @@ std::shared_ptr<Table> CsvParser::parse(const std::string& filename) {
   std::vector<size_t> field_ends;
   while (_find_fields_in_chunk(content_view, *table.get(), field_ends)) {
     // create empty chunk
-    chunks.emplace_back(true);
```

Comment:
This is a good example against the default usage of `emplace_back`. Although `explicit Chunk(const bool has_mvcc_columns);` is marked explicit, `emplace_back` implicitly converts a boolean value to a `Chunk`. Our linter even complains about constructors with a single parameter yet we circumvent it by our always-emplace-back policy.

Could please replace `emplace_back` with `push_back`? :)

Reply:
I decided (and Max didn't fight back) that this is an appropriate usage on `emplace_back`